juju/client/_client8.py

```diff
@@ -877,7 +877,7 @@ class ApplicationFacade(Type):
     async def Get(self, application):
         '''
         application : str
-        Returns -> typing.Union[str, typing.Mapping[str, typing.Any], _ForwardRef('Value')]
+        Returns -> typing.Union[str, typing.Mapping<~KT, +VT_co>[str, typing.Any], _ForwardRef('Value')]
```

Comment:
Do we know what all this is? Is this type definitions for Python being added to the docstrings?